src/components/description/Description.tsx

```diff
@@ -0,0 +1,10 @@
+import { ComponentProps } from "react"
+
+export type TDescription = ComponentProps<'p'> & {size: 'base' | 'sm'} & {color: 'black' | 'gray'}
```

Comment:
Dosent need to intersections ComponentProps<'p'> & {size:string, color:string}

Reply:
resolved